src/lib/common/include/sol-certificate.h

```diff
@@ -74,6 +77,8 @@ void sol_cert_unref(struct sol_cert *cert);
  */
 const char *sol_cert_get_filename(const struct sol_cert *cert);
 
+#endif /*SOL_FEATURE_FILESYSTEM*/
```

Comment:
We only have a implementation for linux so far, that is using filesystem for all methods.